tfrecords/README.md

```diff
@@ -1,2 +1,21 @@
-* `create_tfrecords.py` creates TFRecords with a reduced spatial size so that we don't need to use Dataflow within a Vertex AI pipeline. 
-* `create_tfrecords_fullres.py` creates TFRecords using the original spatial size. 
\ No newline at end of file
+There are two scripts to create TFRecords for [sidewalk dataset](https://huggingface.co/datasets/segments/sidewalk-semantic) hosted in 🤗 Dataset Hub. They are basically same except for one thing. `create_tfrecords.py` creates TFRecords with the features encoded in numeric data type. On the other hand, `create_tfrecords_str.py` creates TFRecords with the features encoded in string data type. 
+- This is due to demonstrate how to create such dataset because string encoding is better in terms of compression.
```

Comment:
```suggestion
- This is to demonstrate how to create such a dataset because string encoding is better in terms of compression.
```

Reply:
Let's also add a note that if the images are serialized as strings, their schema for decoding / parsing needs to change accordingly. [This guide](https://www.tensorflow.org/tutorials/load_data/tfrecord) provides a good overview.